include/saffire/gitversion.h

```diff
@@ -0,0 +1,7 @@
+// Warning: this file is autogenerated. Do not modify.
```

Comment:
I think we should remove this file from the repository and place it in the gitignore as well...